lib/bots/versionbot.ts

```diff
@@ -226,7 +226,7 @@ export class VersionBot extends GithubBot.GithubBot {
                 //  <footer>
                 // As sometimes a development patch may be self-explanatory in the header alone.
                 const lines = commitMessage.split('\n');
-                const lastLine = _.findLastIndex(lines, (line) => { return line.match(/^\s*$/) });
+                const lastLine = _.findLastIndex(lines, (line) => { return line.match(/^\s*$/); });
```

Comment:
Nit: This could use the arrow function expression syntax to drop the brackets/`return`/etc:

```javascript
const lastLine = _.findLastIndex(lines, (line) => line.match(/^\s*$/));
```